base/sink/FairRootFileSink.cxx

```diff
@@ -230,7 +229,10 @@ void FairRootFileSink::Close()
 
 void FairRootFileSink::Reset() {}
 
-void FairRootFileSink::FillEventHeader(FairEventHeader* /* feh */) { return; }
+void FairRootFileSink::FillEventHeader(FairEventHeader* /* feh */)
+{
+    return;
+}
```

Comment:
No need to have a `return` at the end of a function?

```suggestion
{
}
```

Reply:
Done.